<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UnicodeTozh</class>
 <widget class="QDialog" name="UnicodeTozh">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>846</width>
    <height>513</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>240</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>↓Unicode转中文</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>244</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Unicode码对应汉字</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEdit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>74</y>
     <width>681</width>
     <height>161</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Unicode码 \u6587\ub53c</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEdit_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>280</y>
     <width>681</width>
     <height>201</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Times New Roman</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>240</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>↑中文转Unicode</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
